src/components/User/User.tsx

```diff
@@ -0,0 +1,54 @@
+import cn from 'classnames';
+import React from 'react';
+import { Person } from '../../types';
+import { PersonalLink } from '../PersonalLink';
+import { findRelative } from '../../helpers/findRelative';
+
+type Props = {
+  person: Person,
+  people: Person[],
+  selectedUser?: Person,
+};
+
+export const User: React.FC<Props> = ({ person, people, selectedUser }) => {
+  const {
+    sex,
+    born,
+    died,
+    motherName,
+    fatherName,
+    slug,
+  } = person;
+
+  const mother = findRelative('f', person, people);
+  const father = findRelative('m', person, people);
```

Comment:
Better to prepare your people array after you get a response from API. And add `father` and `mother` to every object inside `people` state